doc/changes/unreleased.md

```diff
@@ -6,24 +6,22 @@ T.B.D
 
 T.B.D
 
-### Features
+## Features
 
- - #149: Added Python API for uploading a model to a given bucketfs location
+* #149: Added Python API for uploading a model to a given bucketfs location
 
-### Refactorings
+## Bugfixes
 
- - #201: Added python toolbox to project
- - #294: Improved linter score
- - #305: Updated python toolbox to 1.1.0 & added basic typing checks
- - #295: Activated type check in .pre-commit-config.yaml
- - #296: Activated code formatting in .pre-commit-config.yaml
+* #285: Fixed logic in model output quality checks of udf integration test
+* #285: Deactivated Saas tests because of a broken API
 
-### Bugs
+## Refactorings
 
- - #285: Fixed logic in model output quality checks of udf integration test
- - #285: Deactivated Saas test because of a broken API
-
-### Dependency Updates
-
-- Updated  tornado (6.4.2 -> 6.5.1)
-- #311: Updated transitive dependencies urllib3 (2.4.0 -> 2.5.0) & requests (2.32.3 -> 2.32.4)
+* Updated  tornado (6.4.2 -> 6.5.1)
+* #201: Added python toolbox to project
+* #294: Improved linter score
+* #305: Updated python toolbox to 1.1.0 & added basic typing checks
+* #295: Activated type check in .pre-commit-config.yaml
+* #296: Activated code formatting in .pre-commit-config.yaml
+* #311: Updated transitive dependencies urllib3 (2.4.0 -> 2.5.0) & requests (2.32.3 -> 2.32.4)
+* #317: Added function `download_transformers_model()`
```

Comment:
maybe update this to reflect the actual functions added

Reply:
Uups - you are right. Thanks for this finding!
See next push updating the changelog.